.buildkite/pipeline.yml

```diff
@@ -2,7 +2,13 @@ steps:
   - label: ":shell: Shellcheck"
     plugins:
       - shellcheck#v1.3.0:
-          files: hooks/**
+          files:
+            - hooks/**
+            - commands/**
+            - libs/**
+          options:
+            - "--color=always"
```

Comment:
this is not necessary, it is [always turned on in the plugin](https://github.com/buildkite-plugins/shellcheck-buildkite-plugin/blob/50eda5bb57896d234408d7dcaa9fccc15ff6e953/hooks/command#L99)